src/test/java/org/mule/tools/apikit/MainAppScaffolderOASTest.java

```diff
@@ -68,6 +74,10 @@ public void beforeTest() throws IOException {
 
   @Test
   public void scaffolder() throws Exception {
+    // TODO Get rid of conditional when APIMF-2084 is fixed (4.1.2)
+    if (api.endsWith("AmeriDuo-hplussportsapi-100.json") || api.endsWith("openapi.yaml")) {
```

Comment:
use a descriptive name instead of AmeriDuo-hplussportsapi-100.json

Reply:
Updated